woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -906,7 +906,9 @@ def test_sets_datetime64_dtype_on_init():
         pd.Series(['2020-01-01', None, '2020-01-03'], name=column_name),
         pd.Series(['2020-01-01', np.nan, '2020-01-03'], name=column_name),
         pd.Series(['2020-01-01', pd.NA, '2020-01-03'], name=column_name),
-        pd.Series(['2020-01-01', pd.NaT, '2020-01-03'], name=column_name),
+        # Inferring datetime64[ns] from data containing strings is deprecated
+        # and will be removed in a future version. Only this was a problem.
+        # pd.Series(['2020-01-01', pd.NaT, '2020-01-03'], name=column_name),
```

Comment:
I don't entirely understand why only this Series raised the warning. Not sure if we should find some other way to test inference with `NaT` or ignore for now

Reply:
Can we set the series dtype explicitly here (maybe `object`?) to avoid this warning?

Reply:
object works! thanks for the suggestion!